fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/api/client/keybinding/FabricKeyBinding.java

```diff
@@ -31,6 +32,15 @@ protected FabricKeyBinding(Identifier id, InputUtil.Type type, int code, String
 		super("key." + id.toString().replace(':', '.'), type, code, category);
 	}
 
+	/**
+	 * Returns the configured KeyCode assigned to the KeyBinding from the player's settings.
+	 * @return
+	 */
+	public InputUtil.KeyCode getConfiguredKeyCode()
```

Comment:
This would be better described as `getBoundKeyCode` (or `getBoundKey` following https://github.com/FabricMC/fabric/pull/245#issuecomment-499056099)

Reply:
Should the mixin also be changed to reflect this, or should it stay as `getKeyCode`?